docs/designers-developers/developers/slotfills/main-dashboard-button.md

```diff
@@ -7,12 +7,14 @@ the editor in fullscreen mode.
 
 ```js
 import { registerPlugin } from '@wordpress/plugins';
-import { __experimentalMainDashboardButton } from '@wordpress/edit-site';
+import { 
```

Comment:
It's great that there are docs here for this, just had a request for this! 

Looks like the file just needs to be added to the manifest.json file for it to show up in the block editor handbook.

I've made a PR to publish them - #27317 - if you get a chance to review, @vindl.

Reply:
I was afk for a couple of days so just got to this now. Thanks for the update @talldan!